### ISSUE TITLE
font size issue

### ISSUE THREAD

**Issue by sterding** [winkm89/teachPress#177]
Hi,

I recently noticed that my publication page has a smaller font size for the reference list (but not the abstract).  See the below screenshot. Do you have any idea how to set the font size for the list?

Here is my code in the Wordpress page:
`[tplist order="year ASC, date ASC, title ASC" author_name="simple" author_separator="," show_bibtex=0 show_tags_as="none" show_user_filter=0 size="large" image=”left” image_size=”30″]`

I am using teachPress Version 8.0.2 in Wordpress v5.6.5. Here is the publication list page: https://bioinformatics.bwh.harvard.edu/research/publications/

Thanks,
-Xianjun

![image](https://user-images.githubusercontent.com/682270/133889748-a4ec213b-e029-4e14-aa14-e2fcb255f1b4.png)

**Comment by winkm89** [comment]
Hi, 
I found the bug with the font-size of the abstract. The rest comes from a change in the font-size definition: I'm using now rem font sizes instead of px. This means that the font-size is based on the font-size of the html root element given from your used theme. In you case this root size is: 
```css
html {
font-size: 62.5%;
}
```
And this leads to the small font size.

**Comment by sterding** [comment]
Thanks, winkm89. How can I fix it?

**Comment by sterding** [comment]
I see you fixed it in the new commit. Looking forwards to the new release and testing it!!

**Comment by b-pfl** [comment]
So why does the 2021 template change all font sizes to 0.9rem? 
The font size is clearly too small and from my perspective it should follow the general size definitions of the corresponding theme. At least there should be a switch to disable scaling.

**Comment by sterding** [comment]
I agree. @winkm89, please help to fix this. The reference font is too small to read, very annoying. I am actually thinking to switch to something else for the reference page, if it keeps there.

**Comment by sterding** [comment]
By the way, for other's reference here is how I made it: Login your Wordpress dashboard, Appearance --> Customize --> Additional CSS --> paste the above code there and save. Done.

**Comment by sterding** [comment]
Thanks a lot!!
